x-pack/plugins/monitoring/common/ccs_utils.ts

```diff
@@ -66,28 +66,17 @@ export function prefixIndexPattern(
   }
 
   if (!ccsEnabled || !ccs) {
-    return appendMetricbeatIndex(
```

Comment:
stop appending the `metricbeat-*` pattern

Reply:
I had a thought here that there's a possibility people might be using `monitoring.ui.metricbeat.index` to append custom index patterns. Should we add a deprecation warning if that config key is specified?

Reply:
I don't think we ever supported the user using this as it wasn't documented anywhere and we didn't technically support `metricbeat-*`.  if that's the case would deprecating it be confusing?

Reply:
Ahh, okay if it's not in the docs I guess it's not bad to just let it go away silently.

Reply:
https://github.com/elastic/kibana/issues/120384 has me wondering if we should re-think this approach. I get the feeling there's more than 1 customer problem we've solved by reaching for `monitoring.ui.metricbeat.index`

Reply:
It sounds like the solution would be to not default `ccs` to all (`*`).  Since we are no longer using the metricbeat config value in the code because we are removing it from our queries, it wouldn't do anything to keep it around.

Reply:
Even if we kept the config value there, we aren't using it anymore because we aren't querying metricbeat index anymore.  If we recommended that kind of hack, hopefully we let the user know that.  We probably should have discussed a fix whether we should be setting `ccs` to default to `*`.

Reply:
That's fair. I guess we could say merging this PR raises the importance of having a proper config (which is https://github.com/elastic/kibana/issues/120384)